### ISSUE TITLE
why set a separate distillation token?

### ISSUE THREAD

**Issue by Ga-Lee** [facebookresearch/deit#174]
Hello! Thank you for your wonderful job!
I have a question that when using the convnets as the teacher for distillation, your work chose to set another token(distillation token) to mimic the soft/hard predicts of convnets, and make class token to learn following the labels of dataset, did it?
If so, I wonder why not just make class token to learn from teacher and original label at the same time(using loss function like: loss = alpha * loss_from_label + (1- alpha)* loss_from_kd, only need the class token to cal). Has any experiment been done to check this method when the work was preparing, did it hurt the performance?

Looking forward to your reply!  Thank you a lot !

**Comment by TouvronHugo** [comment]
Hi @Ga-Lee ,
Thanks for your message.
Yes, we try both with and without distillation token (see row 3 and 6 in Table 3). I agree that the caption on this table is not very clear, we need to be updated it.
Best,
Hugo
